src/main/java/com/github/cameltooling/lsp/internal/completion/CamelOptionValuesCompletionsFuture.java

```diff
@@ -76,6 +78,16 @@ public List<CompletionItem> apply(CamelCatalog camelCatalog) {
 				} catch (ApiException e) {
 					LOGGER.error("Error while trying to provide completion for Kubernetes connected mode", e);
 				}
+			} else if(endpointOptionModel.getName().equalsIgnoreCase("lang")
```

Comment:
A good habit to inverse the order to avoid potential NPE
```suggestion
			} else if("lang".equalsIgnoreCase(endpointOptionModel.getName()
```